.github/workflows-source/release-testing.yml

```diff
@@ -182,10 +182,33 @@ jobs:
           cd ci/src/dependencies/
           $SHELL_WRAPPER python3 job/bazel_rust_ic_scanner_release_job.py
 
-  trigger-qualifier-workflow:
-    name: Qualify
-    needs: [ci-main, bazel-system-test-nightly, bazel-system-test-staging, bazel-system-test-hotfix, dependency-scan-release-cut]
-    uses: dfinity/dre/.github/workflows/qualify.yaml@main
-    secrets: inherit
-    with:
-      version: ${{ github.sha }}
+  setup-guest-os-qualification:
+    name: Setting up guest os qualification pipeline
+    <<: *dind-large-setup
+    outputs:
+      matrix: ${{ steps.generate.outputs.output }}
+    steps:
+      - id: generate
+        name: Fetch beginning versions for qualification
+        shell: bash
+        run: |
+          UNIQUE_VERSIONS=$(curl https://rollout-dashboard.ch1-rel1.dfinity.network/api/v1/rollouts | jq -r '.[] | select (.state != "failed") | select (.state != "complete") | .batches | to_entries[] | "\(.value)"' | jq '.subnets[].git_revision' |  sort | uniq | jq -s )
+          echo "Will qualify, starting from versions: ${UNIQUE_VERSIONS}"
+          echo "output=$(jq -cn --argjson versions "$UNIQUE_VERSIONS" '{version: $versions}')" >> $GITHUB_OUTPUT
```

Comment:
Where is `$versions` set? Do you have a test run for this updated workflow where I can see it?

Reply:
In the command: `jq -cn --argjson versions "$UNIQUE_VERSIONS" '{version: $versions}'`
`--argjson versions` creates a `$versions`
Here is the example: https://github.com/dfinity/ic/actions/runs/10724203854